scripts/bootstrap-logkeeper.sh

```diff
@@ -49,4 +49,4 @@ fi
 
 
```

Comment:
i notice a random unused variable called `SPRUCE_DIR` in here 👀 could maybe delete

Reply:
Good eye!